va/va.h

```diff
@@ -4187,6 +4187,26 @@ Synchronization
  * This function blocks until all pending operations on the render target
  * have been completed.  Upon return it is safe to use the render target for a
  * different picture.
+ *
+ * Possible errors:
+ * - \ref VA_STATUS_ERROR_INVALID_CONTEXT: This error occurs when the function is provided with an invalid context.
```

Comment:
> This error occurs when the function is provided with an invalid context.

Function prototype does not have context at all. So, this error is inadequate ( `VA_STATUS_ERROR_INVALID_DISPLAY` seems more reasonable) or at least incorrectly described. `VAStatus vaSyncSurface(VADisplay dpy, VASurfaceID render_target)`.

Reply:
thanks for this advice, the media-driver will return `VA_STATUS_ERROR_INVALID_CONTEXT `instead of `VA_STATUS_ERROR_INVALID_DISPLAY `, so I only correct the comments here to make sure the API compatible with the old version.